src/analytics/formResponseTracker/formResponseTracker.test.ts

```diff
@@ -226,24 +226,27 @@ describe("form with radio buttons", () => {
         "link_path_parts.5": "undefined",
       },
     };
-    expect(instance.pushToDataLayer).toBeCalledWith(dataLayerEvent);
+    expect(BaseTracker.pushToDataLayer).toBeCalledWith(dataLayerEvent);
   });
 
   test("datalayer event should redact information if data is flagged as sensitive", () => {
     const isDataSensitive = true;
-    const instance = new FormResponseTracker(isDataSensitive);
-
-    document.body.innerHTML =
-      '<div id="main-content">' +
-      '<form action="/test-url" method="post">' +
-      "<fieldset>" +
-      "  <legend>test label questions</legend>" +
-      '  <label for="male">test label male</label>' +
-      '  <input type="radio" id="male" name="male" value="Male" checked/>' +
-      '  <label for="female">test label female</label>' +
-      '  <input type="radio" id="female" name="female" value="Male"/>' +
-      "</fieldset>";
-    '  <button id="button" type="submit">submit</button>' + "</form></div>";
+    new FormResponseTracker(isDataSensitive);
+
+    document.body.innerHTML = `
+      <div id="main-content">
+        <form action="/test-url" method="post"> 
+          <fieldset> 
+            <legend>test label questions</legend> 
+            <label for="male">test label male</label> 
+              <input type="radio" id="male" name="male" value="Male" checked/> 
+              <label for="female">test label female</label> 
+              <input type="radio" id="female" name="female" value="Male"/> 
+          </fieldset>
+          <button id="button" type="submit">submit</button>
+        </form>
+      </div>
+    `;
```

Comment:
Yeah I'd say the template literals seem a lot better to me. Less faff with all the double quotes and "+", and it makes it a lot easier to move tags around.

I'm not sure about doing this now though because it will take a while, and I can't find a rule to get `eslint` to enforce it going forwards. `no-useless-concat` doesn't do it because it treats the new-line as a useful form of concatendation. `prefer-template` doesn't seem to do it either and I'm not sure why.